Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.2.1'
+
 #========== GLOBAL GEMS ======================
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.5'
```

Comment:
NB: It's a good idea to upgrade these to the latest stable versions when you get a chance. Lots of bug and security fixes, and performance improvements. Ruby 2.2.4, and Rails 4.2.5.1. We can talk about how to do that if you want.